android/app/src/main/java/com/bluebubbles/messaging/services/FlutterFirebaseMessagingBackgroundExecutor.java

```diff
@@ -184,9 +184,12 @@ public void onMethodCall(MethodCall call, @NonNull Result result) {
      */
     public void startBackgroundIsolate() {
         if (isNotRunning()) {
-            Long callbackHandle = getPluginCallbackHandle();
-            if (callbackHandle != null && callbackHandle != 0) {
+            long callbackHandle = getPluginCallbackHandle();
+            if (callbackHandle != 0) {
+                Log.i(TAG, "Found background isolate callback handle");
                 startBackgroundIsolate(callbackHandle, null);
+            }else {
```

Comment:
Crazy nitpicky, but can u put a space after this braces

Reply:
Done